### DISCUSSION TITLE
How to use Spacy Model in Android (Flutter)

### DISCUSSION THREAD

**Discussion by uzaxirr** [explosion/spaCy#8558] [Help: Coding & Implementations]
Hey,
I want to use spacy model (en_core_web_md) in an android app that is made using flutter.
Is there any way I can use it without serving it as API or without using python execution env in my app, like converting it as tflite model or .h5 file, etc, etc
Please help me out

**Comment by polm** [comment]
spaCy models are not just a bundle of weights, so you can't just convert them to tflite or whatever. Besides the actual model weights they also depend on stuff like the tokenizer that only runs in Python. So I'm not aware of any way you can do this. 

I have heard of people using REST backends with apps but haven't heard of anyone doing anything like what you're requesting.

**Reply by uzaxirr** [reply]
Ohh So i have to serve it as an API then

**Reply by polm** [reply]
That would probably be the easiest way to do it, yes.

**Reply by uzaxirr** [reply]
Any other suggestions for using it on android ??
